### PR TITLE
fix to support loading on nodejs

### LINKED ISSUE #32: React Server Rendering Errors
I'm trying to import this library into a React component which gets server rendered first then registers client side after. So essentially i need the same React component to work in the browser and in Node. This is causing issues in Node because i get `Error: Sorry, the Vimeo Player API is not available in this browser.`.
### Expected Behavior

The Javascript API should allow Node as a browser for some behavior. Even if the API does not support many functions in Node, the API should still allow you to at least import it without errors.
### Actual Behavior

When trying to import this library server side in Node I get `Error: Sorry, the Vimeo Player API is not available in this browser.`
### Steps to Reproduce

Import in any file server side.

### PR DESCRIPTION
**Pull request by colin-tucows**
fix to support loading on nodejs to avoiding error message "Sorry, the Vimeo Player API is not available in this browser."

the fix is based on the branch 'react-server-fix'. my change uses the variable 'isNode' to also skip initializeEmbeds() when loaded on nodejs.

works for my application that uses donejs-ssr, which i believe uses the same concept as react-server.

Fixes #32.

### CODE REVIEW

File: src/lib/compatibility-check.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
+export const isNode = typeof global !== 'undefined' && ({}).toString.call(global) === '[object global]';
 const arrayIndexOfSupport = typeof Array.prototype.indexOf !== 'undefined';
 const postMessageSupport = typeof window.postMessage !== 'undefined';
```
Comment (kavsingh) [inline src/lib/compatibility-check.js]: I have a feeling the `typeof window.postMessage` check will throw if there is now window, causing https://github.com/vimeo/player.js/issues/32#issuecomment-237671072. Probably needs a `typeof window !== undefined` check in there as well?

**Comment by amesas** [comment]
Please, merge this!!!

Moving more and more projects to SSR.

**Comment by alb3rto269** [comment]
Hi,

What is the status of this? 

Is this project still active?

**Comment by dkeulen** [comment]
I would also seriously love this PR to be merged (if the code is approved).
I am having the same issue that i can't even use this library because of no SSR check/support.

**Comment by luwes** [comment]
Sorry for the late response on this. We are looking into this.
I'm going to open a separate PR with the branch `react-server-fix` which my predecessor created.

**Pull request closed without merging**